Issues in kotlin compilation

Is there a problem with the current version of the package? Running the application always return _compileReleaseKotlin FAILED_, debugging the code I get this message:

```bash
> Task :castleio_react-native-castle:compileReleaseKotlin FAILED
e: /home/phelipe/Documentos/temp/TestCastle/node_modules/@castleio/react-native-castle/android/src/main/java/com/reactnativecastle/CastleModule.kt: (26, 35): Unresolved reference: VERSION_NAME

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':castleio_react-native-castle:compileReleaseKotlin'.

```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -32,9 +32,6 @@ android {
   defaultConfig {
     minSdkVersion 21
     targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
-    versionCode 1
-    versionName "1.0"
-
   }
 
   buildTypes {

--- a/android/src/main/java/com/reactnativecastle/CastleModule.kt
+++ b/android/src/main/java/com/reactnativecastle/CastleModule.kt
@@ -22,11 +22,6 @@ class CastleModule(reactContext: ReactApplicationContext) : ReactContextBaseJava
     }
 
     @ReactMethod
-    fun versionString(promise: Promise) {
-      promise.resolve(BuildConfig.VERSION_NAME)
-    }
-
-    @ReactMethod
     fun configure(options: ReadableMap?, promise: Promise) {
       if (options != null) {
         val builder = CastleConfiguration.Builder()

--- a/src/__mocks__/castle.tsx
+++ b/src/__mocks__/castle.tsx
@@ -17,5 +17,4 @@ NativeModules.Castle = {
   userSignature: jest.fn(),
   userAgent: jest.fn(),
   queueSize: jest.fn(),
-  versionString: jest.fn(),
 };

--- a/src/__tests__/index.test.tsx
+++ b/src/__tests__/index.test.tsx
@@ -35,7 +35,6 @@ it(
     '944d7d6c5187cafac297785bbf6de0136a2e10f31788e92b2822f5cfd407fa52'
   )
 );
-it('call .versionString()', testCall('versionString'));
 it('call .clientId()', testCall('clientId'));
 it('call .clientIdHeaderName()', testCall('clientIdHeaderName'));
 it('call .requestTokenHeaderName()', testCall('requestTokenHeaderName'));

